models/plane_lidar/model.config

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<model>
+  <name>plane_lidar</name>
+  <version>1.0</version>
+  <sdf version='1.5'>plane_lidar.sdf</sdf>
+
+  <author>
+   <name>Jaeyoung Lim</name>
+   <email>jaeyoung@auterion.com</email>
+  </author>
+
+  <description>
+    This is a model of a standard plane equipped with a lidar sensor to test terrain following
```

Comment:
Do we need to state it is **to test terrain following**?

Reply:
Maybe its better remove it :smile: